.github/workflows/alert-deploy-arena-adapter.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy alerts to NAIS for Mulighetsrommet-arena-adapter
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - alerts-arena-adapter.yaml
+      - mulighetsrommet-arena-adapter/alerts-arena-adapter.yaml
+      - .github/workflows/alert-deploy-arena-adapter.yaml
+  workflow_dispatch:
+    inputs:
```

Comment:
Det ser ikke ut som at `inputs` ikke referes til i wofklowen, da kan den kanskje slettes helt?

(Det samme gjelder også i .github/workflows/alert-deploy-api.yaml)

Reply:
Med mindre du har planer om å legge til støtte for å kunne deploye til kun valgt miljø via `workflow_dispatch`?

Reply:
Ja, dette var for noe testing jeg drev med. Tar det bort.